openlibrary/i18n/hr/messages.po

```diff
@@ -93,7 +93,7 @@ msgstr "Usmjeri kameru na barkod! 📷"
 #, python-format
 msgid "1 Current Loan"
 msgid_plural "%(count)d Current Loans"
-msgstr[0] "1 trenutačna posudba"
+msgstr[0] "%(count)d trenutačna posudba"
```

Comment:
This needs some testing; I can't recall if the `1` case needs to be without a `%(count)d`.

Reply:
@jimchamp I think this would (correctly) cause your i18n linter to fail!

Reply:
```suggestion
msgstr[0] "1 trenutačna posudba"
```
Yes, this will fail validation. @milotype please revert all translations where `1` is replaced by `%(count)d`.